### PR TITLE
Stop replacing the signature on imported repositories

### PR DESCRIPTION
**Pull request by DrDaveD**

### CODE REVIEW

**Comment by DrDaveD** [comment]
@brianhlin This is very complicated and nobody else has looked at this code, but I need somebody to review it before I can merge.  I think you're the one most likely to be able to understand parts of it (other than perhaps Brian B), but in any case please scan it and if you don't notice any issues give it an approval.

File: install/jumpstart-oasis [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # /bin/bash
+#NOTE: this is no longer maintained nor up to date
```
Comment (brianhlin) [inline install/jumpstart-oasis:2]: Why keep this file around then?

File: install/rebuild-replica [hunk 2]
```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# NOTE: this is no longer maintained or up to date
```
Comment (brianhlin) [inline install/rebuild-replica:2]: Same as above, why keep this around?

**Review by brianhlin — APPROVED** [review]
Minor comments but otherwise looks reasonable to me. Approving

File: install/jumpstart-oasis (hunk 1 above)
Comment (DrDaveD) [reply install/jumpstart-oasis:2]: I think I might still want to use it one day, but the comment is to remind me to update it at that point

File: install/rebuild-replica (hunk 2 above)
Comment (DrDaveD) [reply install/rebuild-replica:2]: again, I might want to use it one day

**Pull request merged**